business_requirement_deliverable_categ/README.rst

```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============================================
+Business Requirement Deliverable Task Category
+==============================================
+
+This module extends module business_requirement_deliverable and business_requirement_projects.
+
+In business requirement, when creating a new resource line for a "Task" resource, you can add the task category.
+
+From the Business Requirement a project with task can be generated based on the resources lines. This module will add the task category from business requirement resource line into the corresponding generated task.
+
+Module business_requirement_deliverable:
+
+* Add field task_categ_id
```

Comment:
Yes, something like
Integrate field task_categ_id from the module project_task_categ